lineman/app/components/invitation_form/invitation_form.haml

```diff
@@ -1,29 +1,30 @@
 .invitation-form
   .lmo-disabled-form{ng-show: 'isDisabled'}
   .modal-header
-    %h3.pull-left {{ 'invitation.invite_people' | translate }}
+    %h1.lmo-h1.modal-title.pull-left {{ 'invitation.invite_people' | translate }}
     %a.close.pull-right{href: '', ng-click: '$close()'} ×
     .clearfix
   .modal-body
-    %h4{translate: 'common.group'}
-    %select.form-control.invitation-form__group-dropdown{ng-model: 'group', ng-required: true, ng-options: 'group as group.fullName() for group in availableGroups()'}
+    %label{for: 'group-dropdown', translate: 'common.group'}
+    %select.invitation-form__group-dropdown.form-control#group-dropdown{ng-model: 'group', ng-required: true, ng-options: 'group as group.fullName() for group in availableGroups()'}
     .lmo-wrap{ng-show: 'group.id'}
-      %h4{translate: 'invitation.instructions'}
+      %label{for: 'invitable-input', translate: 'invitation.instructions'}
       .invitation-form__invitables-input
-        %input.form-control.invitation-form__invitable-input{type: 'text', ng-model: 'fragment', typeahead: 'invitable for invitable in getInvitables()', typeahead-template-url: 'generated/components/invitation_form/invitable_menu.html', typeahead-on-select: 'addInvitation($item)', typeahead-wait-ms: '300'}
+        %input.invitation-form__invitable-input.form-control#invitable-input{type: 'text', ng-model: 'fragment', typeahead: 'invitable for invitable in getInvitables()', typeahead-template-url: 'generated/components/invitation_form/invitable_menu.html', typeahead-on-select: 'addInvitation($item)', typeahead-wait-ms: '300'}
         %loading.invitation-form__loading{ng-show: 'getInvitablesExecuting'}
       %form{name: 'emailValidation'}
         %input.hidden.form-control{type: 'email', ng-model: 'fragment'}
 
     .lmo-wrap{ng-show: 'hasInvitations()'}
-      %h4{translate: 'invitation.to'}
-      .current-invitations{ng-repeat: 'invitation in invitations track by invitation.id'}
+      %strong{translate: 'invitation.to'}
+      .invitation-form__current-invitations{ng-repeat: 'invitation in invitations track by invitation.id'}
         %span {{invitation.name}}
         %span{ng-show: 'invitation.subtitle'} ({{invitation.subtitle}})
-
-      %h4{translate: 'invitation.invitation_message'}
-      %textarea.form-control{ng-model: 'message'}
+      %a{href: '', ng-click: 'isCollapsed = !isCollapsed', ng-show: '!isCollapsed', translate: 'invitation.add_custom_message'}
```

Comment:
Hey from Dolan Springs. We're doing buttons, not `a href=''` now. Please see if you can change this and maybe also update a document in the wiki?